### DISCUSSION TITLE
how about supporting other language except c++

### DISCUSSION THREAD

**Discussion by Viyond** [stephenberry/glaze#19] [Q&A] [answered]
how about supporting other language except c++, such as java, golang, and so on.

**Comment by stephenberry** [comment, marked as answer]
Glaze relies heavily on compile time C++ optimizations and doesn't have a document object model, so it doesn't translate well to other languages. You can use other binding libraries to pass the data to and from Glaze, but direct language support is out of the scope of this library as it would require an entirely different codebase.
